src/SpecsParser.sol

```diff
@@ -24,12 +24,7 @@ library ERC4337SpecsParser {
 
     // Emitted if an invalid storage location is accessed
     error InvalidStorageLocation(
-        address contractAddress,
-        string contractLabel,
-        bytes32 slot,
-        bytes32 previousValue,
-        bytes32 newValue,
```

Comment:
any reason to get rid of these?

Reply:
I don't think theres a way to get the previous value using the new cheatcode (you just read opcodes), we can keep newVakue I guess